src/utils.tsx

```diff
@@ -421,3 +421,7 @@ export const checkForDuplicates = (props: {
 
   return Array.from(duplicateIds);
 };
+
+export const datePickerMinDate = new Date('1900-01-01');
+export const datePickerMaxDate = new Date('2100-01-01');
+export const invalidDateFormatMessage = 'Date format: dd/MM/yyyy';
```

Comment:
I think these should probably be all caps if they are constants, also not sure if utils is the right place for them, I'd be inclined to just put them in the form schemas since it effects the forms, or else perhaps we just need a consts.ts at some point with the `RECENT_MANUFACTURER_CUTOFF_TIME` as well.